google/auth/external_account.py

```diff
@@ -125,6 +130,7 @@ def __init__(
         self._scopes = scopes
         self._default_scopes = default_scopes
         self._workforce_pool_user_project = workforce_pool_user_project
+        self._universe_domain = universe_domain or _DEFAULT_UNIVERSE_DOMAIN
```

Comment:
should be self.universe_domain - right? @sai-sunder-s

Reply:
we made it private for service account for now because adc file doesn't have this field yet

Reply:
Any reason to keep it private here? It defaults to GDU. also the change was made for BYOID configs

Reply:
@viacheslav-rostovtsev

Reply:
I think the reason is universe_domain is not finalized, which needs more discussion 
@TimurSadykov has more context

Reply:
ok lets keep it private